lab-steven/lib/read-files.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const fs = require('fs');
+
+const fileReader = module.exports = function(fileArray, callback) {
+  if (!fileArray) throw new Error('You didn\'t pass in any file paths.');
```

Comment:
You shouldn't throw errors from within your fileReader function. Remember that your callback function takes a error as a parameter. Then in `index.js` line 6 you check if your callback returned an error.